src/utils/dev.js

```diff
@@ -1,118 +1,128 @@
-// reusable code for netlify dev
-// bit of a hasty abstraction but recommended by oclif
 const process = require('process')
 
+const fromEntries = require('@ungap/from-entries')
 const chalk = require('chalk')
 
-const {
-  NETLIFYDEVLOG,
-  // NETLIFYDEVWARN,
-  NETLIFYDEVERR,
-} = require('./logo')
-/**
- * inject environment variables from netlify addons and buildbot
- * into your local dev process.env
- *
- * ```
- * // usage example
- * const { api, site } = this.netlify
- * if (site.id) {
- *   const addonUrls = await addEnvVariables(api, site)
- *   // addonUrls is only for startProxy in netlify dev:index
- * }
- * ```
- */
-const addEnvVariables = async function (api, site) {
-  /** from addons */
-  const addonUrls = {}
-  const addons = await api.listServiceInstancesForSite({ siteId: site.id }).catch((error) => {
-    console.error(error)
-    switch (error.status) {
-      default:
-        console.error(
-          `${NETLIFYDEVERR} Error retrieving addons data for site ${chalk.yellow(
-            site.id,
-          )}. Double-check your login status with 'netlify status' or contact support with details of your error.`,
-        )
-        process.exit()
-    }
-  })
-  if (Array.isArray(addons)) {
-    addons.forEach((addon) => {
-      addonUrls[addon.slug] = `${addon.config.site_url}/.netlify/${addon.slug}`
```

Comment:
Both `addon.slug` and `addon.config.site_url` are `undefined`.
Not sure if this was broken in https://github.com/netlify/cli/pull/1344. I don't think so since the old code was using https://github.com/netlify/js-client/blob/c4a39c22308b60bce8ef1d6c97553bf8a9da56ab/src/addons.js#L30 which is the same as https://open-api.netlify.com/#operation/listServiceInstancesForSite which we are using now.

Fixed in https://github.com/netlify/cli/pull/1490/files#diff-fc861c54030ca23b6d43638aa92f3f446bca1b187ce3b3fea5b8e6d7d1c0a19cR40